easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -86,8 +90,15 @@ def configure_step(self, srcdir=None, builddir=None):
 
         setup_cmake_env(self.toolchain)
 
-        if builddir is None and self.cfg.get('separate_build_dir', False):
-            builddir = os.path.join(self.builddir, 'easybuild_obj')
+        if builddir is None:
+            separate_build_dir = self.cfg['separate_build_dir']
+            if separate_build_dir is None:
+                msg = "Easyconfig parameter 'separate_build_dir' (custom to CMakeMake easyblock) has not set "
```

Comment:
"has not been set " (or "is not set ")

Reply:
I'd rather remove this commit entirely, see reasoning in description
It would also introduce needless depreciation warnings to almost all ecs

Reply:
I removed this commit. It leads to more trouble than it is worth: It would print a deprecation warning for pretty much all EasyConfigs. As mentioned all tested EasyConfigs work with the new default and if a third-party EC fails it is very easy to fix. Having said that: It shouldn't fail in the first place as this would be a sign for something non-standard happening there.